### DISCUSSION TITLE
Interconverting scaffold names and contig names?

### DISCUSSION THREAD

**Discussion by astarr97** [hillerlab/TOGA#169] [Q&A]
Hello,

Thanks for creating this awesome tool and especially the resource with toga annotations for many genomes!  I've been using the annotations available at https://genome.senckenberg.de/download/TOGA/human_hg38_reference/ in conjunction with the 447-way mammalian multi-way alignment from the Zoonomia consortium to study mammalian evolution.  Recently, I was using the annotation for Enhydra_lutris_nereis__sea_otter__enhLutNer1 and noticed that the sequence names did not match between the Zoonomia hal file and the TOGA annotations (e.g. KZ291748.1 in the hal file and QQQE01000479v1 in TOGA).  As far as I can tell, this may be because the hal file is using scaffold names whereas the TOGA annotation file is using contig names.  Is there any way to interconvert to allow use of the TOGA annotation with the Zoonomia hal file?  Any help would be much appreciated and apologies if this has been asked before!

Best, Alex.

**Comment by MichaelHiller** [comment]
Hi Alex, 

enhLutNer1 == GCA_006410715.1 meaning the NCBI GenBank assembly. We rsynced this actually from UCSC. 
Zoonomia may have the corresponding RefSeq assembly and the annoying thing is that NCBI always renames the scaffold names between Genbank and RefSeq, even if the assembly is identical. 

I hope NCBI provides a dictionary to translate the scaff names. Otherwise, it would be worth asking. 

Michael

**Reply by astarr97** [reply]
Thanks so much for the quick reply!  Your answer caused me to search through RefSeq, and after digging some I think Zoonomia used the kenyoni assembly and I should be able to interconvert with the strategy you outlined + will be able to use it moving forward for other species.

Best, Alex.
